### ISSUE TITLE
show measure/bar numbers on input area

### ISSUE THREAD

**Issue by fedelibre** [wbsoft/frescobaldi#988]
I think it may be useful if we could see immediately in Frescobaldi (without any point-and-click or comments) the bar number a line of input belongs to.
Something similar to line number. An user might not be interested in seeing the line number, but rather the measure number.

Probably it should not be the default, but a preference would be nice.

What do you think?

**Comment by uliska** [comment]
That would be cool but is not really achievable yet.

Frescobaldi already tries to determine the timing position in terms of the "moment" of where the cursor is. I'm not sure how reliable that is with differently structured input but for normal documents it seems to work.
So far noone has tried to implement a heuristic to convert that absolute position (e.g. `375/8`) to a real measure because there are lots of loose ends I think, both in terms of plain time changes and in terms of non-standard file organization.

You surely know that this info *is* visible in the status line below the editor window?

---

Apart from that I suggest not to do anything about the area where the line numbers are currently shown (*at the moment*) as we'll soon have to deal with that in the GSoC Git project.

**Comment by fedelibre** [comment]
Yes, I know about the timing position in a measure shown on the status bar. But of course this is a new feature request. Let me add a few arguments in favour of it:

- bar numbers added in a comment are hard to maintain (especially when typesetting a new composition) and can be wrong
- even if point-and-click allows to find the input you want to check, sometimes scrolling the input area to find a certain bar might be quicker

**Comment by uliska** [comment]
I'm not against that at all, I just think it's not within reach yet.

However, *if* we might add such a feature we could also try to add it to the "Goto" dialog so instead of entering a line number you could enter a measure number and part. But that would require even more housekeeping and guesswork.
